src/Http/Headers/src/HeaderNames.cs

```diff
@@ -5,73 +5,170 @@ namespace Microsoft.Net.Http.Headers
+        private readonly static string _ifNoneMatch = "If-None-Match";
+        private readonly static string _ifRange = "If-Range";
+        private readonly static string _ifUnmodifiedSince = "If-Unmodified-Since";
+        private readonly static string _lastModified = "Last-Modified";
+        private readonly static string _location = "Location";
+        private readonly static string _maxForwards = "Max-Forwards";
+        private readonly static string _method = ":method";
+        private readonly static string _origin = "Origin";
+        private readonly static string _path = ":path";
+        private readonly static string _pragma = "Pragma";
+        private readonly static string _proxyAuthenticate = "Proxy-Authenticate";
+        private readonly static string _proxyAuthorization = "Proxy-Authorization";
+        private readonly static string _range = "Range";
+        private readonly static string _referer = "Referer";
+        private readonly static string _retryAfter = "Retry-After";
+        private readonly static string _requestId = "Request-Id";
+        private readonly static string _scheme = ":scheme";
+        private readonly static string _secWebSocketAccept = "Sec-WebSocket-Accept";
+        private readonly static string _secWebSocketKey = "Sec-WebSocket-Key";
+        private readonly static string _secWebSocketProtocol = "Sec-WebSocket-Protocol";
+        private readonly static string _secWebSocketVersion = "Sec-WebSocket-Version";
+        private readonly static string _server = "Server";
+        private readonly static string _setCookie = "Set-Cookie";
+        private readonly static string _status = ":status";
+        private readonly static string _strictTransportSecurity = "Strict-Transport-Security";
+        private readonly static string _tE = "TE";
+        private readonly static string _trailer = "Trailer";
+        private readonly static string _transferEncoding = "Transfer-Encoding";
+        private readonly static string _translate = "Translate";
+        private readonly static string _traceParent = "traceparent";
+        private readonly static string _traceState = "tracestate";
+        private readonly static string _upgrade = "Upgrade";
+        private readonly static string _upgradeInsecureRequests = "Upgrade-Insecure-Requests";
+        private readonly static string _userAgent = "User-Agent";
+        private readonly static string _vary = "Vary";
+        private readonly static string _via = "Via";
+        private readonly static string _warning = "Warning";
+        private readonly static string _webSocketSubProtocols = "Sec-WebSocket-Protocol";
+        private readonly static string _wWWAuthenticate = "WWW-Authenticate";
+        private readonly static string _xFrameOptions = "X-Frame-Options";
+
+        // Use properties rather than constants so ReferenceEquals works
+        public static string Accept => _accept;
```

Comment:
Any particular reason not to just use public static fields directly?

Reply:
🤔 Guess not

Reply:
Or initialise the static properties directly

Reply:
Changed